README.md

```diff
@@ -1,23 +1,28 @@
-# Minikube
+## What is Minikube
```

Comment:
I am using minikube for windows, I have done the setup,but minikube dns is not working.
I did the following steps for minikube setup:

pre-req: configured oracle virtual box
downloaded the minikube version 0.8.0
downloaded kubectl version: 1.3.5
>minikube start
kubectl config set-cluster minikube --server=https://192.168.99.100:30000 --certificate-authority=C:\Users\user\.minikube\ca.crt 
kubectl config set-context minikube --cluster=minikube --user=minikube
kubectl config set-credentials minikube --client-certificate="C:\Users\npulikon\.minikube\apiserver.crt" --client-key="C:\Users\npulikon\.minikube\apiserver.key"

kubectl run -i --tty dnsutils --image=tutum/dnsutils -- sh

Then tried doing nslookup for kubernetes, its not able to resolve.

As I tired deploying yaml files for dns repilca and dns service and its getting automatically deleted.

As Minikube supports Kubernetes features such as:
 +  * DNS
 +  * NodePorts
 +  * ConfigMaps and Secrets
 +  * Dashboards

But for me dns is not working. Can any body please help me out regarding minikube dns?